<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="AryaKS" failures="3" tests="4" name="Pratice Test(failed)" time="50.572" errors="0" timestamp="2020-04-23T19:09:30 NPT">
  <testcase classname="com.pratice.testCases.TC_LoginDDT" name="loginDDT" time="7.32">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.pratice.testCases.TC_LoginDDT.loginDDT(TC_LoginDDT.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.pratice.testCases.TC_LoginDDT" name="loginDDT" time="6.095">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.pratice.testCases.TC_LoginDDT.loginDDT(TC_LoginDDT.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.pratice.testCases.TC_LoginDDT" name="loginDDT" time="6.365">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.pratice.testCases.TC_LoginDDT.loginDDT(TC_LoginDDT.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.pratice.testCases.TC_LoginTest" name="LoginTest" time="2.554"/>
</testsuite> <!-- Pratice Test(failed) -->
